woocommerce/woocommerce
# Add tests for tax display for the shopper

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds a number of end-to-end tests for checking the tax display settings in the store, cart and checkout.

Closes https://github.com/woocommerce/woocommerce-quality/issues/701



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure tests pass in CI
2. Pull this branch
3. `pnpm env:start && pnpm test:e2e-pw shopper/checkout-calculate-tax.spec.js`
4. `pnpm env:restart && pnpm test:e2e-pw`



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
